python-threatexchange/setup.py

```diff
@@ -25,6 +25,12 @@
         "pdqhash>=0.2.2",
         "Pillow",
     ],
+    "pdq_ocr": [
```

Comment:
(Ignorable) This is fine with me. An alternative though would be that if they want to do PDQ_OCR signal types, they should install both the `pdq_hasher` extra and an `ocr` extra. Then this would reduce to just installing `pytesseract`. Might just help cut down a little on the dependency duplication in case you need to update it in the future.